content/docs/2.5/scalers/influxdb.md

```diff
@@ -38,6 +38,7 @@ triggers:
 - `thresholdValue` is provided by the user. This value can vary from use case to use case depending on the data of interest, and is needed to trigger the scaling in/out depending on what value comes back from the query.
 - `query` is the flux query that will yield the value for the scaler to compare the `thresholdValue` against.
 - `metricName` an optional name to assign to the metric. If not set KEDA will generate a name based on masked version of the server url and organization name. If using more than one trigger it is required that all `metricName`(s) be unique.
+- `unsafeSsl` an optional parameter that a user can use when connecting over HTTPS with a self-signed certificate and by default is disabled. 
```

Comment:
Related with [this convention](https://github.com/kedacore/keda-docs/issues/540#issuecomment-930613499)
Could you replace the current line with this new line? (Or something similar, ofc)

- `unsafeSsl` - Skip certificate validation when connecting over HTTPS. (Values: `true`, `false`, Default: `false`, Optional)

The parameter should be like this:
- name - Description. (Values: x, y, z, Default: y, Optional, Extra Info)

Reply:
I replaced with the appropriate description, thanks for the guidance @JorTurFer